.travis.yml

```diff
@@ -1,60 +1,17 @@
+sudo: false
 dist: trusty
 language: rust
-rust:
```

Comment:
I just replaced the build scripts with the ones we used on our latest `master` (which can be found here: https://github.com/szeged/webrender/tree/master_bak_jun2), which we created to ensure that our `webrender` folder builds without any errors.
But since this PR is about transition I think you have the good point, and we should test each feature in isolation as you mentioned above.
I will check if we can go with the original CI scripts without errors, and create an issue on our side and/or send an upstream PR to change the rust version if any error occurs.